iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VariantManagerConstants.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public final class VariantManagerConstants {
+
+    /**
+     * A constant for the id of the initial variant (the one created by default)
+     */
+    public static final String INITIAL_VARIANT_ID = "InitialState";
```

Comment:
here?

Reply:
I did it on purpose because it would break a lot of unit test (outside of powsybl core) as initial state is part of all serialization file.